websites/D/Desmos/presence.ts

```diff
@@ -0,0 +1,167 @@
+const desmosPresence = new Presence({
+	clientId: "886254740399349852" //The client ID of the Application created at https://discordapp.com/developers/applications
+  }),
+  /*strings = desmosPresence.getStrings({
+	play: "presence.playback.playing",
+	pause: "presence.playback.paused"
+	//You can use this to get translated strings in their browser language
+  });*/
+  
+var url: string
+var graphing: number = 0
+var pageType: string
+var title: string = ""
+var numEquations: number = 0
+const startTime: number = Date.now()
+  strings = desmosPresence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+    //You can use this to get translated strings in their browser language
+  });
+
+let url: string,
+ graphing = 0,
+ pageType: string,
+ title = "",
+ numEquations = 0,
+ startTime: number = Date.now();
```

Comment:
startTime never changes, using `const` is more conventional
```suggestion
 const startTime: number = Date.now();
```